README.md

```diff
@@ -29,3 +29,57 @@ will rebuild and reload the app automatically
 $ npm run dev
 ```
 
+## Funding dApps
+
+If this wallet is opened by a dApp, it will accept requests for funds. In order to
+request funds from your dApp, follow these steps:
+
+1. Attach a message event listener to the dApp window
+```js
+window.addEventListener('message', (e) => { /* ... */ });
+```
+2. Open the wallet url in a window from the dApp
+```js
+const walletWindow = window.open(WALLET_URL, 'wallet', 'toolbar=no, location=no, status=no, menubar=no, scrollbars=yes, resizable=yes, width=500, height=600');
+```
+3. Wait for the wallet to load, it will post a `'ready'` message when it's ready to handle requests
+```js
+window.addEventListener('message', (e) => {
+  if (e.data) {
+    switch (e.data.method) {
+      case 'ready': {
+        // ...
+        break;
+      }
+    }
+  }
+});
+```
+4. Send an `'addFunds'` request
+```js
+walletWindow.postMessage({
+  method: 'addFunds',
+  params: {
+    pubkey: /* ... */,
```

Comment:
nit: For these three can you give a concrete example, maybe in the comment, of what an accepted value would look like for a bit more context.